migration-guides/v6.0.0.md

```diff
@@ -193,3 +194,19 @@ const PreviewComponent = ({ file }) => {
 }
 <FileInput previewComponent={PreviewComponent} />
 ```
+
+#### 9. The tag on `<Spinner />` now uses the classname `spinner` in place of an id
+
+`<Spinner />` utilizes a classname and can accept additional custom classes in order to accommodate views where more than one spinner is necessary.
+
+```scss
+// Before
+#spinner {
+  text-align: center;
+}
+
+// After
+.custom-spinner {
+  margin-bottom: 10px;
+}
```

Comment:
I think there might have been a miscommunication. I did not intend for you to also delete the code with `.spinner`. I think you need to show both.